sentences/sv/light_HassLightSet.yaml

```diff
@@ -2,4 +2,11 @@ language: sv
 intents:
   HassLightSet:
     data:
-    - sentences: []
+      - sentences:
+          - "[sätt | vrid | dra [på] | skruva | dimmra] [upp | ner ] <name>[s | n | ns ] [<ljusintensitet>] till <brightness>"
+          - "[sätt | vrid | dra [på] | skruva | dimmra] [upp | ner ] <name>[n]  till <brightness> [<ljusintensitet>]"
+          - "<name>[s | n | ns ] [upp | ner ] [till] <brightness> [<ljusintensitet>]"
+      - sentences:
+          - "[sätt | vrid | dra [på] | skruva | dimmra] [upp | ner ] <alla> <ljuskällor> [<i_på>] <area>[et] [upp | ner ] till <brightness> [<ljusintensitet>]"
```

Comment:
Adding support for:
"sätt all belysning till 50% i garaget"
And maybe extract "[sätt | vrid | dra [på] | skruva | dimmra]" to 
```
expantion_rule: 
    dimma: (sätt | vrid | dra [på] | skruva | dimmra)
```
And use it as: 
```
[<dimma>] [ upp | ner] ......
```

Reply:
Collecting them all under <dimma> sounds good, I will fix that. I was thinking about something similar but couldn’t decide I’m which term to collect them under :)